dev/quickstart/armada-crds-minimal.yaml

```diff
@@ -0,0 +1,170 @@
+apiVersion: install.armadaproject.io/v1alpha1
+kind: ArmadaServer
+metadata:
+  name: armada-server
+  namespace: armada
+spec:
+  pulsarInit: false
+  ingress:
+    ingressClass: "nginx"
+  replicas: 1
+  image:
+    repository: gresearch/armada-server
+    tag: 0.3.103
```

Comment:
Does this need to be updated to a more recent tag? (`v0.4.54` is current latest)

(Actually, if that's updated here, probably the other tag references should be done as well. Otherwise, I'd leave them the same, of course.)

Reply:
I'll check does it run smoothly on v0.4.x